cmd/root.go

```diff
@@ -173,7 +173,6 @@ func init() {
 
 	// Use a proxy or permit a mismatched certificate.
 	rootCmd.PersistentFlags().StringVar(&rest.ProxyAddress, "proxy", "", "The domain name, IP address, or URL of an HTTP proxy server to use")
-	rootCmd.PersistentFlags().BoolVar(&rest.PermitInvalidCertificate, "skip-tls-validate", false, "Skip TLS validation on the connection to the back end")
 	rootCmd.PersistentFlags().MarkHidden("skip-tls-validate")
```

Comment:
Remove MarkHidden along with the flag?

I had to double-check; our tests use --test_only_disable_https, not this flag, so I don't remember why we added it in https://github.com/postmanlabs/observability-cli/pull/195; the comment there suggests a user was blocked (because they had a transparent proxy, perhaps?  In which case --server-tls-name would work.)

Reply:
In 62d3f68.